packages/twenty-server/src/engine/core-modules/auth/controllers/google-apis-auth.controller.ts

```diff
@@ -93,10 +93,11 @@ export class GoogleAPIsAuthController {
           workspaceId,
         );
 
-      await onboardingServiceInstance.skipSyncEmailOnboardingStep(
+      await onboardingServiceInstance.toggleOnboardingConnectAccountCompletion({
```

Comment:
@charlesBochet I find it weird to call it "toggle" looks like more a setter to me since the desired value is expected to be in the input. Toggle usually "flips" the existing value?

Reply:
true